trulens_eval/trulens_eval/keys.py

```diff
@@ -12,16 +12,59 @@
 import logging
 import os
 from pathlib import Path
+import re
+from typing import Any, Optional, Set, Union
 
 import cohere
 import dotenv
 
-from trulens_eval.util import caller_frame
-from trulens_eval.util import UNICODE_CHECK
-from trulens_eval.util import UNICODE_STOP
+from trulens_eval.utils.python import caller_frame
+from trulens_eval.utils.text import UNICODE_CHECK
+from trulens_eval.utils.text import UNICODE_STOP
 
 logger = logging.getLogger(__name__)
 
+# Keep track of values that should not be shown in UI (or added to DB). This set
+# is only used for cases where the name/key for a field is not useful to
+# determine whether it should be redacted.
+values_to_redact = set()
+
+# Regex of keys (into dict/json) that should be redacted.
+RE_KEY_TO_REDACT = re.compile('|'.join([
```

Comment:
Can we be more exhaustive in key checking? User could be using any number of models in their app (Anthropic, Cohere, Bard, ...)

Reply:
Added coverage to the ones I could find, let me know if others are missing.